libs/dmlf/include/dmlf/vm_launcher_interface.hpp

```diff
@@ -55,9 +62,9 @@ class VmLauncherInterface
   virtual bool HasVM(std::string const &name) const                      = 0;
   virtual bool SetVmStdout(std::string const &vmName, VmOutputHandler &) = 0;
 
-  virtual bool CreateState(std::string name)                              = 0;
-  virtual bool HasState(std::string const &name) const                    = 0;
-  virtual bool CopyState(std::string const &srcName, std::string newName) = 0;
+  virtual bool CreateState2(std::string name)                              = 0;
```

Comment:
CreateState2?

Reply:
This class should have been deleted, I removed it (the CreateState2 was an artefact left over from the transition)